Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label. Note: If the checkbox control is not nested in its `label`, wrapping the two elements within another element (such as a `div`) is preferable. Having a shared parent element gives designers more control over the eventual layout of the checkbox group than a `br` does.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
diff --git a/output/Checkbox Group with Validation Error/2/744c64ea-b42b-44a3-bae4-6cd2350e189e.html b/output/Checkbox Group with Validation Error/2/744c64ea-b42b-44a3-bae4-6cd2350e189e.html
index ed0d401..d16612f 100644
--- a/output/Checkbox Group with Validation Error/2/744c64ea-b42b-44a3-bae4-6cd2350e189e.html	
+++ b/output/Checkbox Group with Validation Error/2/744c64ea-b42b-44a3-bae4-6cd2350e189e.html	
@@ -1,10 +1,16 @@
-<fieldset>
+<fieldset aria-describedby="error-message">
   <legend>What fruits do you like?</legend>
-  <input type="checkbox" id="bananas" name="fruits" value="bananas">
-  <label for="bananas">Bananas</label><br>
-  <input type="checkbox" id="oranges" name="fruits" value="oranges">
-  <label for="oranges">Oranges</label><br>
-  <input type="checkbox" id="apples" name="fruits" value="apples">
-  <label for="apples">Apples</label><br>
-  <span style="color: red; display: none;" id="error-message">You must choose one or more fruits</span>
+  <span id="error-message">You must choose one or more fruits</span>
+  <div>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas" aria-invalid="true">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges" aria-invalid="true">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples" aria-invalid="true">
+    <label for="apples">Apples</label>
+  </div>
 </fieldset>
\ No newline at end of file
